[Workers] qr code tutorial revision (#3218)
cloudflare/cloudflare-docs: products/workers/src/content/tutorials/build-a-qr-code-generator/index.md

type = "webpack"
```

> In `index.js`, require the `qr-image` package as the variable `qr`. In the `generate` function, parse the incoming request as JSON, using `request.json` and use the `text` to generate a QR code using `qr.imageSync`:

Reviewer: ```suggestion
In `index.js`, require the `qr-image` package as the variable `qr`. In the `generate` function, parse the incoming request as JSON using `request.json`, and generate a QR code from `text` using `qr.imageSync`:
```
